modules/library/jdbc/src/main/java/org/geotools/jdbc/JDBCFeatureSource.java

```diff
@@ -824,6 +828,7 @@ List<ColumnMetadata> getColumnMetadata(
                 column.nullable = "YES".equalsIgnoreCase(columns.getString("IS_NULLABLE"));
                 column.binding = dialect.getMapping(columns, cx);
                 column.restriction = dialect.getRestrictions(columns, cx);
+                column.remarks = columns.getString(REMARKS);
```

Comment:
You defined setRemarks( String ) but did not use it? Any reason this method is access object fields directly?

Reply:
Good point, was following previous code.  Fixed.